samples/Calling/src/app/App.tsx

```diff
@@ -1,8 +1,9 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { GroupCallLocator, TeamsMeetingLinkLocator } from '@azure/communication-calling';
```

Comment:
Don't we need `GroupCallLocator` & `TeamsMeetingLinkLocator` in the app?

Reply:
Yes, we need for the app. But not for this file. GroupCallLocator & TeamsMeetingLinkLocator are in the union type CallAdapterLocator.